week-1-ts-ecosystem/w01e01-typescript-basics/src/utils.ts

```diff
@@ -0,0 +1,11 @@
+import { User } from './interfaces';
+
+const hasGivenAge = (requiredAge: number): ((user: User) => boolean) => {
+  return (user: User): boolean => user.age >= requiredAge;
+};
+
+export const hasAddress = (user: User): boolean => {
+  return Boolean(user.address);
+};
+
+export const isAdult = hasGivenAge(18);
```

Comment:
Detal: nazwałbym to albo `user.utils.ts` albo jeszcze bardziej precyzyjnie `user.valitators.ts` bo myślę że da się to "zawęzić"

Reply:
poprawione, dzięki!